config/config_helpers_test.go

```diff
@@ -480,6 +481,16 @@ func toStringSlice(t *testing.T, value interface{}) []string {
 	return out
 }
 
+func TestGetPlatform(t *testing.T) {
+	t.Parallel()
+
+	expectedPlatform := runtime.GOOS
+	actualPlatform, err := getPlatform(nil, nil)
+
+	assert.Nil(t, err)
+	assert.Equal(t, expectedPlatform, actualPlatform)
+}
```

Comment:
This unit test doesn't do much, as it's almost a copy/paste of the actual function 😁 

An integration test that uses this helper in a `.hcl` file would be a bit better of a test.

Reply:
You're right!
I'll do better tests!
Thanks for your review and comments!